### DISCUSSION TITLE
changing colors in map statistics

### DISCUSSION THREAD

**Discussion by geir75** [foldynl/QLog#424] [Ideas]
I was just wondering. Would it be possible to change colors in map statistics from yellow to different color to mark confirmed qso?

**Comment by foldynl** [comment]
But it has already been implemented a long time ago.

Without Checking LoTW

![image](https://github.com/user-attachments/assets/a9b79b6a-710d-4991-9138-6cbb25b3d35e)

After checking the LoTW

![image](https://github.com/user-attachments/assets/d3ed93be-ec21-4008-9c61-6b706a261315)

**Reply by geir75** [reply]
OK thank you. I'm new to QLog
